### ISSUE TITLE
Add hyperlinks to abstracts on website

### ISSUE THREAD

**Issue by mjpost** [acl-org/acl-anthology#1714]
For the website, it would be nice to render hyperlinks as hyperlinks, e.g., here: https://aclanthology.org/2020.acl-demos.3/

**Comment by mbollmann** [comment]
This is already supported, the URL only needs to be wrapped in `<url>...</url>` tags in the XML.

**Comment by mbollmann** [comment]
(For example, https://aclanthology.org/2020.acl-main.706/)

**Comment by mjpost** [comment]
Ah, so we must have missed this one at ingestion time. I guess at some point we could update the scripts and apply to old links, but it's going to be low priority. Thanks.

**Comment by mbollmann** [comment]
I was thinking if we had a good URL regex, it might just be a matter of

```bash
sed -i 's#(<abstract>.*[^>])(https?://somethingsomething)#\1<url>\2</url>#' *.xml
```

but I've looked into it briefly and finding a reliable regex for that wasn't so easy. :)

**Comment by mjpost** [comment]
I do have a good URL matcher as [a Python regex](https://github.com/mjpost/sockeye-scripts/blob/master/masking/patterns.url_with_domains.txt).

**Comment by mjpost** [comment]
@xinru1414 can you add this to your queue? It would be nice to find all missing instances in abstracts.

**Comment by xinru1414** [comment]
@mjpost Yes, after #1818

**Comment by mbollmann** [comment]
Ping @GuyAglionby, wrapping URLs in `<url>` tags might be another good addition if you have the time :)
